plugins/de.cognicrypt.core/.classpath

```diff
@@ -8,7 +8,7 @@
 	<classpathentry exported="true" kind="lib" path="lib/commons-cli.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/commons-io.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/commons-lang3.jar"/>
-	<classpathentry exported="true" kind="lib" path="lib/CryptoAnalysis.jar"/>

```

Comment:
Please remove. That would result in errors on other machines.

Reply:
I had forgotten it there. Just removed it.